website/docs/guides/create-indexes/4-create-partial-index.request.js

```diff
@@ -0,0 +1,10 @@
+db.runCommand({
+  createIndexes: 'books',
+  indexes: [
+    {
+      key: { 'availability.format': 1 },
+      name: 'ebook_india_idx',
+      partialFilterExpression: { availability: { $elemMatch: { country: 'India', format: 'E-book' } } }
```

Comment:
Did this query work for you?

```
MongoServerError[CannotCreateIndex]: Error in specification { "key" : { "availability.format" : 1 }, "name" : "ebook_india_idx", "partialFilterExpression" : { "availability" : { "$elemMatch" : { "country" : "India", "format" : "E-book" } } } } :: caused by :: unsupported expression in partial index: availability $elemMatch { "country" : "India", "format" : "E-book" }
```

From [what I can see](https://www.mongodb.com/docs/manual/core/index-partial/#create-a-partial-index), `$elemMatch` is not supported operator for partial indexes.